tensorrt_llm/serve/responses_utils.py

```
@@ -0,0 +1,848 @@
+        conversation, Role.ASSISTANT)
+    return token_ids
+
+
+def parse_output_tokens(tokens: list[int]) -> list[Message]:
+    return get_encoding().parse_messages_from_completion_tokens(
+        tokens, role=Role.ASSISTANT)
+
+
+def parse_output_message(message: Message) -> list[ResponseOutputItem]:
+    """
+    Parse a Harmony message into a list of output response items.
```

Ruff (0.12.2)

One-line docstring should fit on one line

Reformat to one line

(D200)